lib/url.js

```diff
@@ -533,8 +533,9 @@ function autoEscapeStr(rest) {
   // There are ordinary characters at the end.
   if (lastEscapedPos < rest.length)
     return escaped + rest.slice(lastEscapedPos);
-  else  // The last character is escaped.
-    return escaped;
+
```

Comment:
That's the only one, I think, and that one makes sense to me. Will remove it if anyone disagrees.